### PR TITLE
f2c

### PR DESCRIPTION
**Pull request by giacomov**
An historic library to convert F77 programs to C programs, which is still used by some projects in HEP:

http://www.netlib.org/f2c/

### CODE REVIEW

**Comment by giacomov** [comment]
+1

File: recipes/f2c/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{% set version = "1.0.1" %}
+{% set sha256 = "ca404070e9ce0a9aaa6a71fc7d5489d014ade952c5d6de7efb88de8e24f2e8e0" %}
+
+package:
+
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
```
Comment (isuruf) [inline recipes/f2c/meta.yaml]: Can you remove extra spaces like this?

File: recipes/f2c/build.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copy Makefile from the source directory
+
+cp ${RECIPE_DIR}/Makefile .
+
+export LDFLAGS="-Wl,-rpath,${PREFIX}/lib"
```
Comment (isuruf) [inline recipes/f2c/build.sh]: Remove this line. It's set by toolchain

File: recipes/f2c/build.sh [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+case "$(uname)" in
+Linux)
+
+    export CC=${PREFIX}/bin/gcc
```
Comment (isuruf) [inline recipes/f2c/build.sh]: Remove this line. It's set by toolchain

File: recipes/f2c/build.sh [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    
+    make hadd
+    
+    make all
+
+;;
+
+Darwin)
+
+    export CC=clang
```
Comment (isuruf) [inline recipes/f2c/build.sh]: Same here

File: recipes/f2c/build.sh [hunk 5]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copy Makefile from the source directory
+
+cp ${RECIPE_DIR}/Makefile .
```
Comment (isuruf) [inline recipes/f2c/build.sh]: Why? Isn't the Makefile from the source archive enough?

File: recipes/f2c/meta.yaml (hunk 1 above)
Comment (giacomov) [reply recipes/f2c/meta.yaml]: ok

File: recipes/f2c/build.sh (hunk 4 above)
Comment (giacomov) [reply recipes/f2c/build.sh]: ok

File: recipes/f2c/build.sh (hunk 3 above)
Comment (giacomov) [reply recipes/f2c/build.sh]: ok, done

File: recipes/f2c/build.sh (hunk 2 above)
Comment (giacomov) [reply recipes/f2c/build.sh]: ok, done.

File: recipes/f2c/build.sh (hunk 5 above)
Comment (giacomov) [reply recipes/f2c/build.sh]: The makefile in the package does not copy over header files, which are needed by some program to compile against f2c. 

However, following your suggestion, I removed that Makefile. I now copy the one missing header file manually in build.sh.

**Pull request closed without merging**